com.gopeed.Gopeed.yml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gopeed
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  - /git_repo
+  - /com.gopeed.Gopeed.desktop
+modules:
+  - name: gopeed
+    buildsystem: simple
+    build-commands:
+      - cp -r Gopeed /app/
+      - chmod +x /app/Gopeed/gopeed
+      - mkdir -p /app/bin
+      - ln -s /app/Gopeed/gopeed /app/bin/gopeed
+      - install -Dm644 com.gopeed.Gopeed.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 git_repo/ui/flutter/assets/icon/icon_1024.png "${FLATPAK_DEST}/share/icons/hicolor/1024x1024/apps/${FLATPAK_ID}.png"
+    sources:
+      - type: archive
+        dest: Gopeed
+        url: https://github.com/GopeedLab/gopeed/releases/download/v1.4.4/Gopeed-v1.4.4-linux-amd64-portable.tar.gz
+        sha256: B8BD718A2627EC4A13CC7D9634DA1355F30C5EAAD63B3A92707D9A703E1619C5
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/GopeedLab/gopeed/releases/latest
+          tag-query: .tag_name
+          timestamp-query: .published_at
+          version-query: $tag | sub("^[vV]"; "")
+          url-query: '"https://github.com/GopeedLab/gopeed/releases/download/\($tag)/Gopeed-\($tag)-linux-amd64-portable.tar.gz"'
+      - type: git
+        dest: git_repo
+        url: https://github.com/GopeedLab/gopeed
+        tag: v1.4.4
+        commit: 4840c4b8a437c2a01ee4ab68e513dcd305523779
+        x-checker-data:
+          type: git
+          tag-pattern": ^v([0-9.]+)$
```

Comment:
This seems unused?

Reply:
It is used to install the icon.

Reply:
> It is used to install the icon.

You can add a direct link to the icon, instead of the whole repo https://raw.githubusercontent.com/GopeedLab/gopeed/v1.4.4/ui/flutter/assets/icon/icon_1024.png

Reply:
Okay.